src/pages/Login/Login.scss

```
@@ -0,0 +1,31 @@
+.container {
+  background-image: url(../../images/background.jpg);
+  height: 100vh;
+
+}
+
+.field {
+  width: 20rem;
+  height: 17rem;  
+  margin-left: 3rem;
+  margin-top: 14rem;
+  border-radius: 1.8rem;
+  background-color: rgba(255,255,255,0.100);
```

Can you use the variable here? 
```suggestion
  background-color: rgba($variable-name, .1);
```